packages/devextreme/scss/widgets/base/splitter/_index.scss

```diff
@@ -10,3 +11,9 @@
     flex-direction: column;
   }
 }
+
+.dx-resize-handle {
```

Comment:
If you'd like to add this style, let's specify the details for both scenarios

Reply:
i'll remove it for now